src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
```

Comment:
Storing database credentials in the source code is a security risk. It's better to load these values from an external configuration file or environment variables.